server/test/unit/server/webservices/views/test_decorators.py

```diff
@@ -1,14 +1,14 @@
 """
-This module contains tests for the pulp.server.webservices.controllers.decorators module.
+This module contains tests for the pulp.server.webservices.views.decorators module.
 """
 import mock
 
 from .... import base
 from pulp.server.exceptions import PulpCodedAuthenticationException
-from pulp.server.webservices.controllers import decorators
+from pulp.server.webservices.views import decorators
 
 
-class TestAuthenticationMethods(base.PulpWebserviceTests):
+class TestAuthenticationMethods(base.PulpServerTests):
```

Comment:
This object probably would be better named as `PulpWebserviceTests` The entire test suite is about testing Pulp Server, but that object's purpose is to test the webserver aspects of things. Similar updates throughout the PR.